### DISCUSSION TITLE
How to Obfuscated code?

### DISCUSSION THREAD

**Discussion by allran** [cawa-93/vite-electron-builder#198] [Q&A] [answered]
How to obfuscator the project codes when I use vite and typescript. For example, use javascript-obfuscator to obfuscator codes. But i don't find the way to do it, can you help me, thanks。

I try out [javascript-obfuscator](https://github.com/getkey/rollup-plugin-obfuscator), but not success

**Comment by cawa-93** [comment, marked as answer]
Most rollsup's plugins are compatible with Vite. So you can use one of [these](https://www.google.com/search?q=rollup-javascript-obfuscator).

**Reply by allran** [reply]
yes, I use this:https://github.com/getkey/rollup-plugin-obfuscator.git

**Reply by allran** [reply]
I've solved the problem。after use the build.js, load .ts to .js in the dist, then use javascript-obfuscator to set the files.  can't use the plugin in the vite.
